Engine/Engine/Math/Point/point.c

```diff
@@ -0,0 +1,31 @@
+/*
+** EPITECH PROJECT, 2023
+** Title: Engine-3D
+** Author: MasterLaplace
+** Created: 2023-10-12
+** File description:
+** point
+*/
+
+#include "math_point.h"
+#include <stdio.h>
+
+point_f point_f_add(point_f p, point_f w) {
```

Comment:
I believe those functions are defacto inlined by the compiler at this setting of optimization, so maybe adding an `inline` class specifier would be nice for the reader, what do you think?